money-api/src/test/scala/com/comcast/money/api/DefaultSpanHandler.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.api
+
+class DefaultSpanHandler extends SpanHandler {
+
+  override def handle(spanData: SpanData): Unit = println(spanData.toString)
```

Comment:
This is just a sample implementation inside `src/test` for right now, not meant to actually go into production.